Adjust reference to attributes(5) to be attributes(7) in the pkgbuild.1,
pkgtool.1 and spectool.1 man pages.

This patch should be sent upstream.

--- pkgbuild-1.3.105/doc/pkgbuild.1.orig	2016-04-20 09:23:29.152475566 -0700
+++ pkgbuild-1.3.105/doc/pkgbuild.1	2016-04-20 09:23:46.151702633 -0700
@@ -152,7 +152,7 @@
 
 .SH ATTRIBUTES
 
-See \fBattributes\fR(5) for descriptions of the following attributes:
+See \fBattributes\fR(7) for descriptions of the following attributes:
 
 .TS
 box;
@@ -169,7 +169,7 @@
 
 Spec file description in /usr/share/doc/pkgbuild/spec-files.txt
 
-\fBattributes\fR(5), \fBpkgtool\fR(1), \fBspectool\fR(1), \fBpkg\fR(5), \fBpkgmk\fR(1)
+\fBattributes\fR(7), \fBpkgtool\fR(1), \fBspectool\fR(1), \fBpkg\fR(5), \fBpkgmk\fR(1)
 
 .SH NOTES
 
--- pkgbuild-1.3.105/doc/pkgtool.1.orig	2016-04-20 09:48:22.091081082 -0700
+++ pkgbuild-1.3.105/doc/pkgtool.1	2016-04-20 09:48:45.699934158 -0700
@@ -264,7 +264,7 @@
 
 .SH ATTRIBUTES
 
-See \fBattributes\fR(5) for descriptions of the following attributes:
+See \fBattributes\fR(7) for descriptions of the following attributes:
 
 .TS
 box;
@@ -281,7 +281,7 @@
 
 Spec file description in /usr/share/doc/pkgbuild/spec-files.txt
 
-\fBattributes\fR(5), \fBpkgbuild\fR(1), \fBspectool\fR(1), \fBpkg\fR(5), \fBpkgmk\fR(1)
+\fBattributes\fR(7), \fBpkgbuild\fR(1), \fBspectool\fR(1), \fBpkg\fR(5), \fBpkgmk\fR(1)
 
 .SH NOTES
 
--- pkgbuild-1.3.105/doc/spectool.1.orig	2016-04-20 09:48:09.082323294 -0700
+++ pkgbuild-1.3.105/doc/spectool.1	2016-04-20 09:48:56.338792549 -0700
@@ -209,7 +209,7 @@
 
 .SH ATTRIBUTES
 
-See \fBattributes\fR(5) for descriptions of the following attributes:
+See \fBattributes\fR(7) for descriptions of the following attributes:
 
 .TS
 box;
@@ -226,7 +226,7 @@
 
 Spec file description in /usr/share/doc/pkgbuild/spec-files.txt
 
-\fBattributes\fR(5), \fBpkgbuild\fR(1), \fBpkgtool\fR(1), \fBpkg\fR(5), \fBpkgmk\fR(1)
+\fBattributes\fR(7), \fBpkgbuild\fR(1), \fBpkgtool\fR(1), \fBpkg\fR(5), \fBpkgmk\fR(1)
 
 .SH NOTES
 
